internal/util/reftracker/radoswrapper/radoswrapper.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2022 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package radoswrapper
+
+import (
+	"github.com/ceph/go-ceph/rados"
+)
+
+type (
+	IOContext struct {
+		*rados.IOContext
+	}
+
+	WriteOp struct {
+		IoCtx *rados.IOContext
+		*rados.WriteOp
+	}
+
+	ReadOp struct {
+		IoCtx *rados.IOContext
+		*rados.ReadOp
+	}
+
+	ReadOpOmapGetValsByKeysStep struct {
+		*rados.ReadOpOmapGetValsByKeysStep
+	}
+)
+
+var _ IOContextW = &IOContext{}
+
+func NewIOContext(ioctx *rados.IOContext) IOContextW {
+	return &IOContext{
+		IOContext: ioctx,
+	}
+}
+
+func (c *IOContext) GetLastVersion() (uint64, error) {
```

Comment:
Need comments for all the exported methods.

Reply:
I'll add comments in the respective interfaces instead of implementations. Let me know if you want me to add comments to implementations instead.

Reply:
Yes should be fine.